providers/gitlab.go

```diff
@@ -7,12 +7,21 @@ func init() {
 		DisplayName: "GitLab",
 		AuthType:    Oauth2,
 		BaseURL:     "https://gitlab.com",
-		Oauth2Opts: &Oauth2Opts{
-			GrantType:                 AuthorizationCode,
-			AuthURL:                   "https://gitlab.com/oauth/authorize",
-			TokenURL:                  "https://gitlab.com/oauth/token",
-			ExplicitScopesRequired:    false,
-			ExplicitWorkspaceRequired: false,
+		CustomOpts: &CustomAuthOpts{
+			Headers: []CustomAuthHeader{
+				{
+					Name:          "PRIVATE-TOKEN",
```

Comment:
@Jkarage previously you had "PRIVATE TOKEN" (no dash) which is not a valid header value